README.md

```diff
@@ -39,28 +39,28 @@ Inspired by Doomspork's [Elixir companies][elixir-companies] list.
 * [Cachix](https://cachix.org/) ([GitHub](https://github.com/cachix/elm2nix)) -
   Build Nix packages once and share them for good.
 * [Concourse CI](https://concourse-ci.org)
-  ([Github](https://github.com/concourse/concourse/)) - Concourse CI is a
+  ([GitHub](https://github.com/concourse/concourse/)) - Concourse CI is a
```

Comment:
Fixed spelling of Github (most links already use correct spelling with uppercase H, just fixed the ones that had lowercase h).